schema/atac_schema.md

```diff
@@ -0,0 +1,473 @@
+## scATAC-seq assay types
+
+<i>paired assay</i>: any descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0010891"><code>"EFO:0010891"</code></a> for <i>scATAC-seq</i> that is also a descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008913"><code>"EFO:0008913"</code></a> for <i>single-cell RNA sequencing</i>
+
```

Comment:
paired is effectively only "10x multiome" ? I'm trying to understand the rationale for writing the general definition. For example, are there pending NTR(s) for other assays that would meet the requirement?

Reply:
No specific NTR issues open, but this is future-proofing. And we would quickly go to EFO with suggestions to group more under "single-cell RNA sequencing" to ensure future NTRs get links appropriately.

Reply:
Is there the possibility that an unexpected term might be added causing validation issues - a'la _One of These Things (Is Not Like the Others)_. ? I guess we could simply include it in the review of EFO updates. Just thinking out loud.

Reply:
We wouldn't accept a new assay without reviewing it in EFO or adding it there ourselves. So the edge case would be a term we have reviewed and accepted later gets the ontology links updated to where it suddenly fits the criteria. So possible? yes. Though seems unlikely.